fab/inventory/icds

```diff
@@ -48,11 +79,17 @@ web0
 [webworkers:children]
 web0
 web1
+web2
+web3
+web4
+web5
 kafka0
 
 [postgresql:children]
 pg0
 pg1
+pg2
+pg3
```

Comment:
How should the shards be updated [here](https://github.com/dimagi/commcarehq-ansible/blob/master/ansible/vars/icds/icds_public.yml#L81)?